.github/workflows/unit-tests.yaml

```
@@ -0,0 +1,51 @@
+name: Unit Tests
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    types:
+    - opened
+    - edited
+    - synchronize
+jobs:
+  unit_tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    # Cache
+    - uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+
+    # Setup
+      - name: Set up Nodejs 16
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
```

```suggestion
    - name: Set up Nodejs 16
      uses: actions/setup-node@v1
      with:
        node-version: 16
```
I think I messed the identation